src/main/resources/static/css/forms.css

```diff
@@ -17,4 +17,14 @@
 
 .logout-form {
     background-color: transparent;
+}
+
+.footer {
```

Comment:
as mentioned - footer is very broad and used everywhere so it's nice to put it in styles.css